### DISCUSSION TITLE
Why do you use alis?

### DISCUSSION THREAD

**Discussion by picodotdev** [picodotdev/alis#150] [General]
Hello archers,

Arch Linux provides in the official installation medium a guided installer [archinstall](https://github.com/archlinux/archinstall) as addition to the default method with the [installation guide](https://wiki.archlinux.org/index.php/Installation_guide).

I would like to know:

* If you use alis even so, what do you like about alis to prefer it over archinstall?
* If you use archinstall, what is the killer feature to use archinstall instead alis?

All feedback is good and will be considerated to improve alis.

**Comment by funcrab** [comment]
archinstall:
- official: https://github.com/archlinux/archinstall
  - Included in the official ISO (but not supported by the official Arch community)
- script: python
  - Simple functions but complex scripts
  - Difficult to get 1:1 correspondence with manual operation
  - Difficult to compare with official archwiki procedures
- config: json
  - Highly scalable, but complex and hard to read

alis:
- official: https://github.com/picodotdev/alis
  - Not included in the official ISO, and is not supported by the official Arch community
  - Basically follows ArchWiki, so it's easy to maintain on your own
  - There is a good chance that legitimate feedback will be incorporated
- script: bash
    * Rich functions but simple script
    * Easy to customize
    * Easy to make 1:1 correspondence between manual operation and automatic execution format
    * Easy comparison with official archwiki procedures
- config: bash variable definition
    * Simple format with only variable specification
